DBZ-6721 Add docs for SMTs (#5634)
debezium/debezium: documentation/modules/ROOT/pages/connectors/vitess.adoc

}
----

[[vitess-efficient-transaction-metadata]]
=== Efficient Transaction Metadata

> If you enable the connector to provide transaction metadata, it generates significantly more data, including the number of messages sent, and the size of each message.

Reviewer: ```suggestion
If you enable the connector to provide transaction metadata, it generates significantly more data, including information about the number and size of the messages that the connector sends.
```

> If you enable the connector to provide transaction metadata, it generates significantly more data, including the number of messages sent, and the size of each message.

Author: ```suggestion
If you enable the connector to provide transaction metadata, it generates significantly more data. It will send additional messages to the transaction topics and the size of each message sent to the data change topics will be increased (because it includes the transaction metadata block).
```
